### PR TITLE
Added primary key fields to API endpoints

### PR DESCRIPTION
**Pull request by Rashmil-1999**

### CODE REVIEW

**Comment by Rashmil-1999** [comment]
@pujanm  please review it and merge it

File: BEProjectsApp/serializers.py [hunk 1]
```diff
@@ -20,16 +21,17 @@ class Meta:
...
-    # teacher = serializers.HyperlinkedIdentityField(
-    #     many=False, view_name="BEProjectsApp:teacherprofile-detail", read_only=True
-    # )
+    teacher = serializers.HyperlinkedIdentityField(
+        many=False, view_name="BEProjectsApp:teacherprofile-detail", read_only=True
+    )
     contributor = serializers.HyperlinkedRelatedField(
         many=True, view_name="BEProjectsApp:contributor-detail", read_only=True
     )
     # contributers = ContributerSerializer(many=True, read_only=True)
```
Comment (pujanm) [inline BEProjectsApp/serializers.py]: Remove this comment if not required.

**Comment by pujanm** [comment]
@Rashmil-1999 Did you make any changes on the front side?

**Comment by pujanm** [comment]
@Rashmil-1999 build failed. I guess you forgot to run `black`.

**Pull request merged**